wpilibj/src/main/java/edu/wpi/first/wpilibj/DriverStation.java

```diff
@@ -1042,12 +1042,12 @@ public static void waitForData() {
    * <p>Checks if new control data has arrived since the last waitForData call on the current
    * thread. If new data has not arrived, returns immediately.
    *
-   * @param timeout The maximum time in seconds to wait.
+   * @param timeoutSeconds The maximum time in seconds to wait.
    * @return true if there is new data, otherwise false
    */
-  public static boolean waitForData(double timeout) {
-    long startTime = RobotController.getFPGATime();
-    long timeoutMicros = (long) (timeout * 1000000);
+  public static boolean waitForData(double timeoutSeconds) {
+    long startTimeMicroS = RobotController.getFPGATime();
+    long timeoutMicroS = (long) (timeoutSeconds * 1000000);
```

Comment:
any chance to put this in E notation? I find that it's more readable, the team might disagree with me though.